lib/gui/app/pages/main/controllers/flash.js

```diff
@@ -81,6 +158,11 @@ module.exports = function (
         $state.go('success')
       }
     }).catch((error) => {
+      // When flashing is cancelled before starting above there is no error
+      if (!error) {
```

Comment:
I'm going to add this to the list of problems one wouldn't have without Promises ;P